ISHPermissionKit/Requests/ISHPermissionRequestSiri.m

```diff
@@ -0,0 +1,48 @@
+//
+//  ISHPermissionRequestSiri.m
+//  ISHPermissionKit
+//
+//  Created by Hagi on 24/08/16.
+//  Copyright © 2016 iosphere GmbH. All rights reserved.
+//
+
+#import "ISHPermissionRequestSiri.h"
+#import "ISHPermissionRequest+Private.h"
+#import <Intents/Intents.h>
+
+@implementation ISHPermissionRequestSiri
+
+- (ISHPermissionState)permissionState {
+    return [self permissionStateForSiriState:[INPreferences siriAuthorizationStatus]];
+}
+
+- (ISHPermissionState)permissionStateForSiriState:(INSiriAuthorizationStatus)siriState {
+    switch (siriState) {
+        case INSiriAuthorizationStatusAuthorized:
+            return ISHPermissionStateAuthorized;
+
+        case INSiriAuthorizationStatusDenied:
+        case INSiriAuthorizationStatusRestricted:
+            return ISHPermissionStateDenied;
+
+        case INSiriAuthorizationStatusNotDetermined:
+            return ISHPermissionStateUnknown;
+    }
+
+    NSAssert(NO, @"Undefined permission state: %@", @(siriState));
+    return [self internalPermissionState];
```

Comment:
I don't see a benefit in making this a class method as it's only called from instance methods, and it could bite us later. This is also how other requests do it.